pkg/logs/internal/tailers/file/rotate_windows.go

```diff
@@ -8,12 +8,36 @@
 
 package file
 
-import (
-	"os"
-)
+import "github.com/DataDog/datadog-agent/pkg/util/log"
+
+// DidRotate returns true if the file has been log-rotated.
+//
+// On Windows, log rotation is identified by the file size being smaller
+// than the last offset read.
+func (t *Tailer) DidRotate() (bool, error) {
+	f, err := openFile(t.fullpath)
+	if err != nil {
+		return false, err
+	}
+	defer f.Close()
+
+	st, err := f.Stat()
+	if err != nil {
+		log.Debugf("Error stat()ing file %v", err)
```

Comment:
```suggestion
		log.Debugf("Error calling stat() on file %v", err)
```
or maybe
`stat failed on file`?